src/sbvr-api/sbvr-utils.coffee

```diff
@@ -823,19 +824,22 @@ exports.handleODataRequest = handleODataRequest = (req, res, next) ->
 		console.error('An error occured while constructing the response', e, e.stack)
 		res.sendStatus(500)
 
+getErrorResponseBody = (e) ->
+	return e.payload or e.message
```

Comment:
If the primary use is to be used as the response body, would it make sense to name it as `body`?